eo-maven-plugin/src/test/java/org/eolang/maven/AssembleMojoTest.java

```diff
@@ -41,6 +42,7 @@
 public final class AssembleMojoTest {
 
     @Test
+    @Disabled
```

Comment:
@marceloamadeu I believe this test fails because you always redefine field in `PullMojo`
And the fake objectionary from line 70 gets erased